core/io/file_access_pack.cpp

```diff
@@ -260,7 +260,7 @@ Ref<FileAccess> PackedSourcePCK::get_file(const String &p_path, PackedData::Pack
 //////////////////////////////////////////////////////////////////
 
 Error FileAccessPack::open_internal(const String &p_path, int p_mode_flags) {
-	ERR_FAIL_V(ERR_UNAVAILABLE);
+	ERR_PRINT("Can't open pack-referenced file.");
```

Comment:
What's "pack-referenced"? Isn't this just a low-level handler for Godot PCK files?

Reply:
I just copied the error text from here:

https://github.com/godotengine/godot/blob/36bedd341a3f9e1a335a795cfb4b3c73f18903a2/core/io/file_access_pack.cpp#L373-L376

I think the only purpose of using `ERR_FAIL_V` here is to print the error to the console, since `ERR_UNAVAILABLE` is always returned. But a clear message is better.